hack/run-functional-tests.sh

```diff
@@ -82,22 +83,22 @@ run_resource_tests() {
   # grab local resource from ${TESTS_MANIFEST}
   local resource
   
-  resource=$(yq read --stripComments "${TESTS_MANIFEST}" "resources.[${resource_index}].kind")
+  resource=$(yq r "${TESTS_MANIFEST}" "resources.[${resource_index}].kind")
```

Comment:
I think you want `yq "resources.[${resource_index}].kind" "${TESTS_MANIFEST}"` here and all related places ;)

Reply:
I have been a little bit busy last week and didn’t tried to figure out what is happening. But thank you so much for your comment about how to solve that last step to our PR. 

Today I will finish it and hope every thing goes fine.

Reply:
I think in early versions of yq v4.x.x there was a brief time where "e"/"eval" was not assumed as the default command. So maybe changing changing each instance of `yq` to `yq eval` explicitly will help resolve this. I"ve tested on another project that uses this same functional testing framework and it seems to have worked fine.

Reply:
I agree. I'm looking into what we can do about that. We can't easily use the normal `set -e` for BASH since some of these tests pass by failing as expected.

Reply:
Sorry guys @phenixblue @awoimbee , I have been very busy with another internal project. But I hope I'll start over again and fix that last issue asap.